mail-client/alpine/alpine-2.21.1-r1.ebuild

```diff
@@ -7,8 +7,7 @@ inherit autotools toolchain-funcs vcs-snapshot
 DESCRIPTION="An easy to use text-based based mail and news client"
 HOMEPAGE="http://www.washington.edu/alpine/ http://repo.or.cz/alpine.git/"
 GIT_COMMIT="843b2f16abfd949e09b1c5465387b1b0f724994a"
-SRC_URI="http://repo.or.cz/alpine.git/snapshot/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
-RESTRICT="mirror"
+SRC_URI="http://repo.or.cz/alpine.git/snapshot/${GIT_COMMIT}.tar.gz -> ${PF}.tar.gz"
```

Comment:
PF is no good here. It means that when someone revbumps it (e.g. due to dep change), he'll suddenly change filename for no reason.

Also, please match the upstream name, i.e. `alpine-843b2f1` (grab it off `${GIT_COMMIT::n}` where *n* is some number), and use that both for `->` name and `S`; then remove vcs-snapshot as that eclass is deprecated.

Reply:
Thanks for pointing this out, I fixed it accordingly.